backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -603,13 +603,12 @@ public void setResizable(boolean resizable) {
 	public void setVSync (boolean vsync) {
 	}
 
-	/** Sets the preferred framerate for the application. Default is 60.
+	/** Sets the preferred framerate for the application. Default is 60. Usage is not advised on mobile platforms.

```

Comment:
I think this comment should go on the `Graphics` interface instead (I would say "Usage is not **generally** advised" on mobile backends" or something on the lines of, only use if you know what you're doing).